cool-core/src/main/java/com/nus/cool/core/schema/Codec.java

```diff
@@ -66,6 +66,11 @@ public enum Codec {
    */ 
   Range,
 
+  /**
+   * 
+   */
+  Set,
```

Comment:
Codec.Range determines whether it is range type or set type.
Because there only two types, so one support type is either set or range. In this situation, one flag is enough.

But It should have two flag to identify the type of data

Reply:
Got it.